samples/Samples.Amazon.SQS/Properties/launchSettings.json

```diff
@@ -0,0 +1,20 @@
+{
+  "profiles": {
+    "Samples.AwsSqs": {
```

Comment:
nit: The name of this profile doesn't match the name of the sample program `Samples.Amazon.SQS`